registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/MosipEventBusFactory.java

```diff
@@ -68,4 +69,24 @@ public Tracing getTracing() {
     public void setTracing(Tracing tracing) {
         this.tracing = tracing;
     }
+    
+    public String getKafkaBootstrapServers() {
```

Comment:
Since environment.getProperty() is dynamic, this can be evaluated after the startup only. Creating a PropertiesUtil, in which we will check for null for the property and throw exception accordingly.